scripts/binary_version_header.py

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+import ctypes
+import hashlib
+import binascii
+
+MAGIC = "$B!N"
+VERSION = [0x01,0x02]
+VERSION_OFFSET = 4
+
+_fields_V1 = [
```

Comment:
Why are there V1 and V2 versions already, has this already used in some product somewhere? (not asking which/where)